src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java

```diff
@@ -281,6 +281,14 @@ LEFT JOIN (SELECT DISTINCT dataset_id AS id FROM dar_dataset) dar_ds_ids ON dar_
     """)
     List<Dataset> getActiveDatasets();
 
+    @SqlQuery("""
+        SELECT dp.property_value
```

Comment:
nit: adding DISTINCT will offload the work of set coercion to the db
```suggestion
        SELECT DISTINCT dp.property_value
```